test/integration/openid4ci_test.go

```diff
@@ -109,11 +114,36 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 		require.NoError(t, err)
 		require.Contains(t, vc.VC.Issuer.ID, tc.issuerDIDMethod)
 
-		subID, err := verifiable.SubjectID(vc.VC.Subject)
-		require.Contains(t, subID, didID)
+		if !tc.isSD {
+			subID, err := verifiable.SubjectID(vc.VC.Subject)
+			require.NoError(t, err)
+			require.Contains(t, subID, didID)
+			checkActivityLogAfterOpenID4CIFlow(t, activityLogger, tc.issuerProfileID, subID)
```

Comment:
Can we put this validation outside the if-else condition ?

Reply:
it will not work, because we use selective disclosure there will not be an `id` inside credentialSubject. So it will not appear in the logs.

Reply:
Ok. Do we need a follow up PR to fix the activity log for SD ?